bridges/centralized-ethereum/src/actors/dr_database.rs

```diff
@@ -35,14 +42,38 @@ pub enum DrState {
     Finished,
 }
 
+impl Default for DrState {
+    fn default() -> Self {
+        Self::New
+    }
+}
+
 /// Make actor from DrDatabase
 impl Actor for DrDatabase {
     /// Every actor has to provide execution Context in which it can run.
     type Context = Context<Self>;
 
     /// Method to be executed when the actor is started
-    fn started(&mut self, _ctx: &mut Self::Context) {
-        log::debug!("DrReporter actor has been started!");
+    fn started(&mut self, ctx: &mut Self::Context) {
+        log::debug!("DrDatabase actor has been started!");
+
+        let fut = storage_mngr::get::<_, DrDatabase>(&BRIDGE_DB_KEY)
+            .into_actor(self)
+            .map(
+                |dr_database_from_storage, act, _| match dr_database_from_storage {
+                    Ok(dr_database_from_storage) => {
+                        if let Some(dr_database_from_storage) = dr_database_from_storage {
+                            act.dr = dr_database_from_storage.dr;
+                            act.max_dr_id = dr_database_from_storage.max_dr_id;
+                        }
+                    }
+                    Err(e) => {
+                        panic!("Error while getting bridge database from storage: {}", e);
+                    }
+                },
+            );
+
+        ctx.spawn(fut);
```

Comment:
To avoid handling other messages before the database is initialized:

```suggestion
        ctx.wait(fut);
```